### PR TITLE
Adding Quasar Chain

### PR DESCRIPTION
**Pull request by arhamchordia**

### CODE REVIEW

File: apps/quasar/app.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "appCategory": "Asset Management",
```
Comment (delivan) [inline apps/quasar/app.json:2]: "appCategory" must be one of [DeFi, Social, NFT, DAO, Tool, Liquid Staking]

File: apps/quasar/app.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "appName": "Quasar",
+  "appSummary": "Quasar is an appchain for IBC-enabled vaults that automatically execute customizable financial strategies across independent blockchains.",
+  "appWebsiteUrl": "https://www.quasar.fi/",
```
Comment (delivan) [inline apps/quasar/app.json:5]: we recommend adding a link to your website where users can try your app right away.
```suggestion
  "appWebsiteUrl": "https://app.quasar.fi/",
```

**Review by delivan — CHANGES_REQUESTED** [review]

File: apps/quasar/app.json (hunk 1 above)
Comment (arhamchordia) [reply apps/quasar/app.json:2]: Sure. Will update it soon.

Comment (arhamchordia) [reply apps/quasar/app.json]: Update to DeFi

**Review by delivan — APPROVED** [review]

**Pull request merged**